### ISSUE TITLE
Neos is unstable, while using a MouthTracker without an Eye Tracker on latest build

### ISSUE THREAD

**Issue by ProbablePrime** [Neos-Metaverse/NeosPublic#1918]
## Describe the bug
Recently(2021.3.31.528), has been proving unstable for me. I spend 3-15 minutes in the game and then Neos freezes with "Waiting" from SteamVR. 

This issue seems to be exacerbated by other users being present. Dropping that down to 3-7 Minutes

## Relevant issues
I can find many issues relating to instability but I feel this one may be different due to the logs

## To Reproduce
I don't have any I'm sorry.

## Expected behavior
I'd love to play Neos for 6-8 Hours like Normal :)

## Bug information (please complete the following information):
 - How often does it happen: Sometimes
 - Does the bug persist after restarting Neos? Yes
 - Neos Version: 2021.3.31.528
 - Neos Platform: Windows
 - Regression: Spent about 6 hours in 2021.3.29.523 without issues

## Additional context
Logs \o/:
[ALFRED - 2021.3.31.528 - 2021-03-31 02_56_06.log](https://github.com/Neos-Metaverse/NeosPublic/files/6235787/ALFRED.-.2021.3.31.528.-.2021-03-31.02_56_06.log)
[ALFRED - 2021.3.31.528 - 2021-03-31 02_55_23.log](https://github.com/Neos-Metaverse/NeosPublic/files/6235789/ALFRED.-.2021.3.31.528.-.2021-03-31.02_55_23.log)

```
`02:58:02.187 ( 90 FPS)	Error initializing SRanipal Eye Tracking: NOT_SUPPORT_EYE_TRACKING

   at void BaseX.UniLog.Error(string message, bool stackTrace)
   at IEnumerator ViveProEyeTrackingDriver.InitializeAPI(bool isFirstInitialization, bool isLip)+MoveNext()`
```

Seems to be being spammed. Perhaps an un-handled error?

## Reporters:
Me, Cheese

**Comment by Frooxius** [comment]
I'll add handling for this use-case in the next build, which will stop initializing the eye tracking after a bit. Unfortunately this is a problem with the SR_Anipal runtime/SDK, trying to initialize it has a certain chance of a crash, so when initializing eye tracking, it will only try to initialize lip tracking once and then re-initialize only a few times (same with eye tracking if it gets disconnected).

Currently that's tuned for eye tracking, without lip tracking though, but not the other way around. In the meanwhile, I recommend just reboot or just restarting SR_Anipal runtime. We haven't changed anything with it in the last build, so this is most likely just a random occurrence that you're getting the issue now.

**Comment by ProbablePrime** [comment]
Thanks, I'm a tad confused by your second paragraph though.

The wording is a little hard to understand, are you saying that your next build will fix my issue or that we'll need to wait for a SR_Anipal update to fix the issue?

Either is totally fine i'll just make sure to restart SR_Anipal when this happens for now or at least until you respond.

Thank you!!

**Comment by Frooxius** [comment]
Next build should fix your issue by giving up on trying to initialize the eye tracking and not making SR_Anipal crash by the repeated attempts.

**Comment by ProbablePrime** [comment]
Perfect, thank you so much!

**Comment by Frooxius** [comment]
Pushed fix in 2021.4.2.690, hopefully this will help improve the stability on your end (and others who use the tracker without eye tracking)!
